modules/client_bottommenu/bottommenu.lua

```diff
@@ -496,17 +508,45 @@ function onClickOnNextCalendar()
     reloadEventsSchedulerCurrentPage()
 end
 
-function Booster_creature(data)
-    if modules.game_things.isLoaded() then
-        local creatureraceid = modules.game_cyclopedia.RACE[data.creatureraceid]
-        local bossraceid = modules.game_cyclopedia.RACE_Bosstiary[data.bossraceid]
-        if creatureraceid then
-            creature_boosted:setOutfit({type=creatureraceid.type})
-            creature_boosted:getCreature():setStaticWalking(1000)
-        end
-        if bossraceid then
-            boss_boosted:setOutfit({type=bossraceid.type})
-            boss_boosted:getCreature():setStaticWalking(1000)
-        end
+-- (internal)
+-- set creature/boss to boosted slot
+local function applyToBoostedSlot(raceId, outfitWidget, imageWidget, fileName)
+    -- check if raceId was provided in the JSON response
+    if not raceId then
+        return
+    end
+
+    -- fetch race data
+    local raceData = g_things.getRaceData(raceId)
+
+    -- check if race id is present in the staticdata
+    if raceData.raceId == 0 then
+        local msg = string.format("[%s] Creature with race id %s was not found.", fileName, data.creatureraceid)
```

Comment:
Have you tested when the raceis not found? This will throw an error since data is not defined.

Reply:
> it returns an empty struct

For me it was throwing an error saying data is nil...

haha